samples/discoverNatBehavior.c

```diff
@@ -2,108 +2,115 @@
 
 #define NETWORK_INTERFACE_NAME_PARAM "-i"
 #define STUN_HOSTNAME_PARAM          "-s"
-#define DEFAULT_STUN_HOST            "stun:stun.sipgate.net:3478"
 
 BOOL filterFunc(UINT64 data, PCHAR name)
 {
     CHAR* desiredInterface = (CHAR*) data;
-    if (desiredInterface == NULL || STRCMP(name, desiredInterface) == 0) {
+    if (desiredInterface == NULL || STRNCMP(name, desiredInterface, STRLEN(name)) == 0) {
```

Comment:
Since we're switching `STRCMP` --> `STRNCMP` we should also use `STRNLEN` instead of `STRLEN`

Reply:
Done!

Reply:
Got the length from https://unix.stackexchange.com/a/532650